### PR TITLE
LIMS-5369 : archive suborder from random order 

### PR DESCRIPTION
**Pull request by modeso-NahlaMakhlouf**
Description : orders :Make sure that by clicking on Archive from Suborder options a confirmation popup will appear and user can Archive this suborder with its corresponding analysis

Local execution results
`C:\Users\ADMIN\PycharmProjects\1lims-automation>nosetests -vs --nologcapture --tc-file=config.ini ui_testing/testcases/basic_tests/test006_orders.py:OrdersTestCases.test070_archive_subord
er
C:\Users\ADMIN\AppData\Local\Programs\Python\Python38\lib\site-packages\nose\plugins\manager.py:394: RuntimeWarning: Unable to load plugin noseprogressive = noseprogressive:ProgressivePlu
gin: No module named '_curses'
  warn("Unable to load plugin %s: %s" % (ep, e),
2020-08-27 15:49:19.678 | INFO     | api_testing.apis.base_api:_get_authorized_session:46 - Get authorized api session.
2020-08-27 15:49:19.681 | INFO     | api_testing.apis.base_api:_get_authorized_session:47 - islam:islam
2020-08-27 15:49:20.362 | INFO     | api_testing.apis.base_api:_get_authorized_session:58 - session ID : To2POZacosCQ0ZaLI85W_qIEMmR0LLBntIQlENSSHas .....

DevTools listening on ws://127.0.0.1:52298/devtools/browser/e7c9f7a9-b5f6-4d2a-8820-a535f4f89246
orders :Make sure that by clicking on Archive from Suborder options a confirmation popup will appear ...
2020-08-27 15:50:07.533 | INFO     | ui_testing.testcases.base_test:setUp:27 - Test case : test070_archive_suborder
2020-08-27 15:50:12.400 | INFO     | ui_testing.pages.base_pages:wait_until_page_is_loaded:573 - wait until page is loaded
2020-08-27 15:50:12.453 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-27 15:50:13.270 | INFO     | api_testing.apis.orders_api:set_configuration:395 - set order configuration
2020-08-27 15:50:13.994 | INFO     | api_testing.apis.base_api:set_configuration:73 - status code: 1
2020-08-27 15:50:14.001 | INFO     | ui_testing.testcases.basic_tests.test006_orders:test070_archive_suborder:2306 - select random order
2020-08-27 15:50:14.083 | INFO     | ui_testing.testcases.basic_tests.test006_orders:test070_archive_suborder:2308 - open child table
2020-08-27 15:50:14.282 | DEBUG    | ui_testing.pages.base_pages:sleep_medium:47 - wait up to 2 sec
2020-08-27 15:50:17.759 | INFO     | ui_testing.testcases.basic_tests.test006_orders:test070_archive_suborder:2310 - archive suborder from orders active table
[<selenium.webdriver.remote.webelement.WebElement (session="9a92d03c4510d977095a67a3e3cde516", element="c1040443-c6e6-4a5b-aa9a-6b2272d5efa2")>, <selenium.webdriver.remote.webelement.WebE
lement (session="9a92d03c4510d977095a67a3e3cde516", element="76a5c763-b35f-4f73-beab-a4f2d36bec58")>]
[{'': '', 'Material Type': 'Raw Material', 'Article Name': 'Eier roh 11-20', 'Article No.': 'Pigment 2', 'Test Date': '26.08.2020', 'pickingTime': '-', 'Test Plans': '03f1482155', 'Change
d On': '27.08.2020', 'Departments': '-', 'Changed By': 'islam', 'Attachments': '-', 'comment': '-', 'manfactureDate': '-', 'expirationDate': '-', 'Chargen Nr': '-', 'Options': '', 'Time D
ifference': '0', 'Analysis No.': "3'600-2'020", 'Analysis Results': '-', 'Test Units': '-', 'Status': 'Open', 'Container Nr.': '-', 'Forwarding': '', 'Monitoring Bereich': '-', 'interne L
ieferschein Nr.': '-', 'sample': '-', 'externes Labor': '-', 'Validation date': '-', 'Tank': '-', 'Validation by': '-', 'Lauf Nr.': '-', 'Report sent by': '-', 'Nachkontrolle': '-', 'Ship
ment Date': '26.08.2020', 'amount': '-', 'casNr': '-', 'iupac': '-', 'noOfSamples': '-', 'Mehrfachauswahl': '-', 'Num': '-'}]
2020-08-27 15:50:21.603 | INFO     | ui_testing.pages.base_pages:open_row_options:283 - open record options menu
2020-08-27 15:50:22.029 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-27 15:50:23.299 | INFO     | ui_testing.pages.base_pages:confirm_popup:76 - confirming the popup
2020-08-27 15:50:24.048 | DEBUG    | ui_testing.pages.base_pages:sleep_small:43 - wait up to 1 sec
2020-08-27 15:50:29.766 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-27 15:50:31.240 | INFO     | ui_testing.testcases.basic_tests.test006_orders:test070_archive_suborder:2322 - make sure that suborder is archived
2020-08-27 15:50:31.845 | INFO     | ui_testing.pages.orders_page:filter_by_analysis_number:142 - Filter by analysis number : 3'600-2'020
2020-08-27 15:50:34.724 | INFO     | ui_testing.pages.base_pages:wait_until_page_is_loaded:573 - wait until page is loaded
2020-08-27 15:50:34.770 | DEBUG    | ui_testing.pages.base_pages:sleep_tiny:39 - wait up to 0.5 sec
2020-08-27 15:50:35.891 | DEBUG    | ui_testing.pages.base_pages:sleep_medium:47 - wait up to 2 sec
2020-08-27 15:50:39.947 | INFO     | ui_testing.testcases.base_test:tearDown:33 - go to dashboard page
2020-08-27 15:50:41.269 | INFO     | ui_testing.testcases.base_test:tearDown:35 - TearDown.     
ok

----------------------------------------------------------------------
Ran 1 test in 83.048s

OK
`

### CODE REVIEW

File: ui_testing/testcases/basic_tests/test006_orders.py [hunk 1]
```diff
@@ -1,2298 +1,2328 @@
...
+    #         self.assertEqual(testplan_name, testplans_testunits_names_in_popup[1]['test_plan'])
+    #         self.info("assert that the test plan I added in the test plan popup ".format(testplan_name,
+    #                                                                                      testplans_testunits_names_in_popup[
+    #                                                                                          1]['test_units'][0]))
+    #         self.assertEqual(testunit_name, testplans_testunits_names_in_popup[1]['test_units'][0])
+    #
+    #     @skip("https://modeso.atlassian.net/browse/LIMSA-127")
+    #     def test062_testplans_popup_after_edit_by_replace(self):
+    #         """
+    #         Orders: Test plan: Test
```
Comment (ghada-37) [inline ui_testing/testcases/basic_tests/test006_orders.py:2311]: you don't used this, so no need to get it

Comment (ghada-37) [inline ui_testing/testcases/basic_tests/test006_orders.py:2314]: remove any print please, if you need to print any data use self.info() instead

**Review by ghada-37 — CHANGES_REQUESTED** [review]

File: ui_testing/testcases/basic_tests/test006_orders.py (hunk 1 above)
Comment (modeso-NahlaMakhlouf) [reply ui_testing/testcases/basic_tests/test006_orders.py:2314]: done

Comment (modeso-NahlaMakhlouf) [reply ui_testing/testcases/basic_tests/test006_orders.py:2311]: done

File: .github/workflows/execution.sh [hunk 2]
```diff
@@ -12,7 +12,7 @@ EXECUTION_FILES=(
     ui_testing/testcases/header_tests/test011_rolesandpermissions.py
   )
 
-TEST_REG='test[0-9]{3}'
+TEST_REG='test070'
```
Comment (ghada-37) [inline .github/workflows/execution.sh:15]: you don't run test case, please update  EXECUTION_FILES to select order file only

**Review by ghada-37 — CHANGES_REQUESTED** [review]

**Comment by ghada-37** [comment]
After you update Excution File, check if this test case fails or not and tell me please ?
As I think it will fail due to this bug https://modeso.atlassian.net/browse/LIMSA-280

File: .github/workflows/execution.sh (hunk 2 above)
Comment (modeso-NahlaMakhlouf) [reply .github/workflows/execution.sh:8]: done

**Comment by modeso-NahlaMakhlouf** [comment]
> After you update Excution File, check if this test case fails or not and tell me please ?
> As I think it will fail due to this bug https://modeso.atlassian.net/browse/LIMSA-280

>@ghada : it passed

**Review by ghada-37 — CHANGES_REQUESTED** [review]

**Comment by ghada-37** [comment]
@NahlaMakhlouf I found test case merged into master called archive suborder. 
It only don't assert that pop up  appears when you archive suborder. 
So, could you please create new branch and use function on master,  just add check of pop up to it. 
And close this PR. 
https://github.com/modeso-open-source/1lims-automation/blob/f6e56c3e7e8fdcb03d4be06f0e5190912a415551/ui_testing/testcases/basic_tests/test006_orders.py#L948

File: ui_testing/testcases/basic_tests/test006_orders.py [hunk 3]
```diff
@@ -1,2298 +1,2326 @@
...
```
[the remark this answers is not in the document]
Comment (modeso-NahlaMakhlouf) [reply ui_testing/testcases/basic_tests/test006_orders.py]: done

**Comment by ghada-37** [comment]
> > After you update Excution File, check if this test case fails or not and tell me please ?
> > As I think it will fail due to this bug https://modeso.atlassian.net/browse/LIMSA-280
> 
> > @Ghada : it passed

It will pass if selected order has only one suborder, in case of multiple suborders it will fail

**Pull request closed without merging**